project_description/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===================
+Project Description
+===================
+
+This module extends the functionality of projects to support setting a
+description for each project and allow you to search by that description.
+
+Usage
+=====
+
+To set a description for a project, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Edit or create one.
+#. You will see a new *Description* tab, with an HTML area to set the project's description.
+
+.. figure:: static/description/image.png
+   :alt: Project form with the description tab
+
+To search for a project by its description, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Enter any word in the search box.
+#. Select *Search Description*.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/11.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Tecnativa (https://www.tecnativa.com): Rafael Blasco <rafael.blasco@tecnativa.com>
```

Comment:
Contributors name should be `name` and `Email`.

Ex. Rafael Blasco <rafael.blasco@tecnativa.com>

Reply:
See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
Yes, but It's a Good way to add only `name` and `Email`.

Because a company name is Email Domin so.